### DISCUSSION TITLE
Is there a way to hide casts to 0x0 bytes?

### DISCUSSION THREAD

**Discussion by f-raZ0R** [NationalSecurityAgency/ghidra#7280] [Q&A] [answered]
A lot of times, a check for whether a struct exists/isn't empty will be with a `cmp` or equivalent instruction, against 0.
This will be, intuitively translated as "if struct == 0", or "if struct == null". However, ghidra will still show the cast, making it busily show up as "if struct == (Struct* ) 0x0". Is there a way to get rid of these casts, so the code looks cleaner?

**Comment by astrelsky** [comment]
There is an option to use NULL instead. It's somewhere in the decompiler settings.

**Reply by f-raZ0R** [reply]
What's the option called? Can't find it

**Reply by astrelsky** [reply, marked as answer]
![image](https://github.com/user-attachments/assets/10f417ca-cff3-4e8a-8471-474c14d0bec5)

**Reply by f-raZ0R** [reply]
Thank you!
A bit weird that it uses the color for variables instead of the color for constants, though...
